lighthouse-core/audits/dobetterweb/link-blocking-first-paint.js

```diff
@@ -55,31 +56,42 @@ class LinkBlockingFirstPaintAudit extends Audit {
       };
     }
 
-    let totalSpendTime = 0;
-    const results = artifact.items
-      .filter(item => item.tag.tagName === tagFilter)
-      .map(item => {
-        totalSpendTime += item.spendTime;
+    const filtered = artifact.filter(item => item.tag.tagName === tagFilter);
 
-        return {
-          url: item.tag.url,
-          label: `delayed first paint by ${item.spendTime}ms`
-        };
-      });
+    const startTime = filtered.reduce((t, item) => Math.min(t, item.startTime), Number.MAX_VALUE);
```

Comment:
FWIW, I wrote some quick code to calculate the blocking length from the union of these requests and I could find no page where the two numbers differed (no site had a break between overlapping requests). It's possible I didn't test enough bad sites, though :)